src/azul/service/service_config/request_config.json

```diff
@@ -175,6 +175,7 @@
     "instrumentManufacturerModel",
     "pairedEnd",
     "project",
+    "projectId",
```

Comment:
Doesn't this have the effect of adding yet another aggregation?

Reply:
Unnecessary new facet "projectId" removed